index.html

```diff
@@ -5,11 +5,11 @@
   <meta charset="UTF-8" />
   <!-- i have set the viewport  -->
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <!-- i am changing the title to improve seo -->
```

Comment:
No I meant, like this -> `Changed the title to improve seo` ; omit the " I " part.

Reply:
ok

Reply:
thanks for the appreciation.  And thanks to you for all the guidance :)